scripts/build_bytebase_docker.sh

```diff
@@ -11,14 +11,17 @@ cd "$(dirname "$0")/../"
 . ./scripts/build_init.sh
 
 echo "Start building Bytebase docker image ${VERSION}..."
-
-docker build -f ./scripts/Dockerfile \
+# need docker buildx
+docker buildx build -f ./scripts/Dockerfile \
+    --platform=linux/amd64,linux/arm64 \
     --build-arg VERSION=${VERSION} \
     --build-arg GO_VERSION="$(go version)" \
     --build-arg GIT_COMMIT="$(git rev-parse HEAD)"\
     --build-arg BUILD_TIME="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"  \
     --build-arg BUILD_USER="$(id -u -n)" \
-    -t bytebase/bytebase .
+    --sbom=false \
+    --push \
+    -t bytebase/bytebase:${VERSION} .
```

Comment:
We don't use this script to push to docker hub.

Reply:
The content of this PR will have an impact on your releases, so you need to handle it carefully. Additionally, since I'm not very familiar with your CI process, it's best for your team to perform the necessary fixes.